docs/guides/configuration.rst

```diff
@@ -54,7 +56,8 @@ default values
         "inMemory": false,
         "loglevel": "warn",
         "url": "",
-        "adminPassword": ""
+        "adminPassword": "",
+        "name": "hoodie"
```

Comment:
maybe use `"my-hoodie-app"` as an example value? `"hoodie"` might be confusing

Reply:
Definitely agree :sunglasses:! Thanks man.